web/cobrands/greenwich/layout.scss

```diff
@@ -257,3 +374,156 @@ body.twothirdswidthpage .content .sticky-sidebar {
+                        background-repeat: no-repeat;
+                    }
+                    .visually-hidden {
+                        border: 0;
+                        clip: rect(0 0 0 0);
+                        height: 1px;
+                        margin: -1px;
+                        overflow: hidden;
+                        padding: 0;
+                        position: absolute;
+                        width: 1px;
+                    }
+                }
+            }
+        }
+        .site-footer__block {
+            display: inline-block;
+            float: left;
+        }
+        .site-footer__copyright-links {
+            padding-left: 3rem;
+            // width: 58.33333%;
+            text-align: left;
+            .site-footer__links {
+                .list--footer-nav {
+                    margin: 0 0 0.5rem;
+                    .list__item {
+                        display: inline-block;
+                        padding-right: 1.125rem;
+                        a {
+                            &:visited {
+                                color: $contrast_text;
+                            }
+                        }
+                    }
+                }
+            }
+            .site-footer__information {
+                .site-footer__rights {
+                    font-size: 1.375rem;
+                    margin-bottom: 0;
+                }
+                .site-footer__address {
+                    font-size: 1rem;
+                    font-weight: 300;
+                    margin: 0.125rem 0 0.375rem;
+                }
+                .site-footer__suppliers {
+                    font-size: 1rem;
+                    font-weight: 300;
+                    margin-bottom: 0;
+                }
+            }
+        }
+    }
+}
+
+.societyworks-footer {
+    padding: 2em 0;
+    background: #2e3033;
+    color: rgba(255, 255, 255, 0.6);
+    .container {
+        p {
+            margin: 0;
+        }
+    }
+    .societyworks-footer__logo {
```

Comment:
Since this cobrand only includes a single logo in the societyworks footer, I‘d be tempted to combine these two `.societyworks-footer__logo` and `.societyworks-footer__logo--fms-pro` rulesets.